content/en/docs/ibm/create-cluster.md

```diff
@@ -44,12 +44,28 @@ Get the Kubeconfig file:
 ibmcloud ks cluster config --cluster $CLUSTER_NAME
 ```
 
-From here on, please see [Install Kubeflow](/docs/ibm/deploy/install-kubeflow).
+From here on, please see [Install Kubeflow](/docs/ibm/deploy/install-kubeflow-on-iks).
 
 
 ## Create and setup a new cluster
 
-Follow these steps to create and setup a new [IBM Cloud Kubernetes Service(IKS) cluster:
+Follow these steps to create and setup a new IBM Cloud Kubernetes Service(IKS) cluster:
+
+### Choose between a `classic` or `vpc-gen2` provider.
+A `vpc-gen2` cluster does not expose each node to the public internet directly and thus has more secure
+and more complex network setup. A `classic` provider, does expose each cluster node to the public internet, and thus has
+simpler networking setup - it might be a good option for trial run of Kubeflow.
+
+However, in the case of `classic` provider set-up is easier, the onus to secure each service that is exposed through
+Kubernetes (e.g. NodePort/Load Balancer) is on the user. To secure the cluster access, one can either configure
+authentication for each service exposed or setup firewall rules for each open port. Configuring firewalls for `classic`
+provider is covered in this doc
+[IBM Cloud doc link](https://cloud.ibm.com/docs/containers?topic=containers-firewall#firewall_outbound), configuring
+authentication is covered in the [install-Kubeflow](/docs/ibm/deploy/install-kubeflow-on-iks#multi-user-auth-enabled)
+section.
+
+Considering the risk associated with exposing computing resources on the public internet, it might be easier to setup
+`vpc-gen2` cluster than securing a `classic` provider cluster.
```

Comment:
```suggestion
Considering the potential risk associated with exposing computing resources on the public internet, you may find it easier to set up
a `vpc-gen2` cluster rather than securing a `classic` provider cluster.
```